### PR TITLE
chore(sass): Load custom styles properly

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: dd72aad4217706626d0a0c6d83c26195341e77f2
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: dd72aad4217706626d0a0c6d83c26195341e77f2
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: dd72aad4217706626d0a0c6d83c26195341e77f2
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by pmuir** [comment]
Fixes #243

File: src/assets/stylesheets/shared/_custom-mixins.scss [hunk 1]
```diff
@@ -0,0 +1,99 @@
+@import 'bootstrap/mixins';
```
Comment (joshuawilson) [inline src/assets/stylesheets/shared/_custom-mixins.scss]: How does it find "bootstrap" and "patternfly"? My IDE can't identify the source for them.

Comment (pmuir) [inline src/assets/stylesheets/shared/_custom-mixins.scss]: The includePaths are setup in webpack.common

Comment (mitchharpur) [inline src/assets/stylesheets/shared/_custom-mixins.scss]: as i understand from the code there is a webpack plugin that appends the relative path to the bootstrap sass as it walks the dependency tree

Comment (pmuir) [inline src/assets/stylesheets/shared/_custom-mixins.scss]: No, this is actually just standard node-sass/libsass includePaths

**Pull request merged**